.buildkite/nightly.yml

```diff
@@ -39,14 +39,15 @@ steps:
       system: x86_64-linux
       queue: benchmark
 
-  - label: 'Database Migrations Test'
-    commands:
-      - "rm -rf state-migration-test*"
-      - "nix-build nix/migration-tests.nix -o migration-tests"
-      - "./migration-tests/runall.sh"
-    timeout_in_minutes: 60
-    agents:
-      system: x86_64-linux
+  # TODO: ADP-548 Port migrations test to shelley
```

Comment:
Does this mean that we'll be without migration tests for a while?

If so, I think it would be worth making a ticket just for that (if there isn't one already), and upping the priority.

Reply:
OK the new ticket is ADP-549.